Frontend-Development/README.md

```diff
@@ -1,15 +1,22 @@
 # Frontend Roadmap
+
 ## Introduction
+
 ### What is front end developer
+
 - [What is a frontend developer? - frontendmasters.com](https://frontendmasters.com/guides/front-end-handbook/2018/what-is-a-FD.html)
 - [Frontend web developer - Mozilla Developer docs](https://developer.mozilla.org/en-US/docs/Learn/Front-end_web_developer)
 - [What is frontend development - Freecodecamp](https://www.freecodecamp.org/news/front-end-developer-what-is-front-end-development-explained-in-plain-english/)
+
 ### What is frontend web development
+
 - [Frontend Development explained - Mayuko(Youtube)](https://www.youtube.com/watch?v=qyHyFsT7Hig)
 - [Frontend web development - Wikipedia](https://en.wikipedia.org/wiki/Front-end_web_development)
 
 ## Understanding Internet
+
```

Comment:
Remove the whitespace

Reply:
sir which whitespaces you are talking about ? Can you give some examples?